src/objects/imageButton.js

```diff
@@ -1,11 +1,5 @@
-let imageButtons = [];
-
-const ImageButton = class
-{
-	img;
-	x;
-	y;
-	callback;
+// TODO: Add some documentation here.
```

Comment:
Especially helpful here would be some documentation around the `x` and `y` variables, because it's not immediately clear what they _mean_ - I understand they're coordinates, but I'm not exactly sure how to use them. Should they be the coordinates of the top-left of the image? Or where we want to place them in the canvas? Or something else?

Reply:
It uses p5 coordinates of which are top left origin. I should have punched that in while writing the class.